pineappl_py/pineappl/grid.py

```diff
@@ -298,6 +298,66 @@ def convolve_with_two(
         )
 
     def evolve(
+        self,
+        operators,
+        mur2_grid,
+        alphas_values,
+        lumi_id_types="pdg_mc_ids",
+        order_mask=(),
+        xi=(1.0, 1.0),
+    ):
+        """Create an FKTable with the EKO.
+
+        Convenience wrapper for :meth:`pineappl.pineappl.PyGrid.evolve()`.
+
+        Parameters
+        ----------
+        operators : dict
+            EKO Output
+        mur2_grid : list[float]
+            renormalization scales
+        alphas_values : list[float]
+            alpha_s values associated to the renormalization scales
+        lumi_id_types : str
+            kind of lumi types (e.g. "pdg_mc_ids" for flavor basis, "evol"
+            for evolution basis)
+        order_mask : list(bool)
+            Mask for selecting specific orders. The value `True` means the corresponding order
+            is included. An empty list corresponds to all orders being enabled.
+        xi : (float, float)
+            A tuple with the scale variation factors that should be used.
+            The first entry of a tuple corresponds to the variation of
+            the renormalization scale, the second entry to the variation of the factorization
+            scale. If only results for the central scale are need the tuple should be
+            `(1.0, 1.0)`.
+
+        Returns
+        ------
+        PyFkTable :
+            raw grid as an FKTable
+        """
+        operator_grid = np.array(
+            [op["operators"] for op in operators["Q2grid"].values()]
+        )
+        q2grid = list(operators["Q2grid"].keys())
+        return FkTable(
+            self.raw.evolve(
+                np.array(operator_grid),
+                operators["q2_ref"],
+                np.array(operators["inputpids"], dtype=np.int32),
+                np.array(operators["inputgrid"]),
+                np.array(q2grid, dtype=np.float64),
+                np.array(operators["targetpids"], dtype=np.int32),
+                np.array(operators["targetgrid"]),
+                np.array(mur2_grid, dtype=np.float64),
+                np.array(alphas_values, dtype=np.float64),
+                xi,
+                lumi_id_types,
+                np.array(order_mask, dtype=bool),
+            )
+        )
+
+    def evolve2(
```

Comment:
I guess you want to prepend the `2` also to the Python call below

Reply:
Yes we want that, good catch!

Reply:
Fixed in commit 6081b979ff2b0448797ced6a5beedd267d8ceb6a.